### DISCUSSION TITLE
如何@别人

### DISCUSSION THREAD

**Discussion by Mreate** [Mrs4s/go-cqhttp#2173] [Q&A] [answered]
如何在群里发送消息时引用(回复)一条消息？如何@别人？

**Comment by xiangxiangxiong9** [comment, marked as answer]
[CQ:at,qq=QQ号]
若采用数组传递，则加入 {"type": "at", “data": {"qq": "QQ号"}}
[将QQ号替换为实际的QQ号，如要@全体，则替换为 all ]
